### ISSUE TITLE
[NON-ISSUE] Cross validating training result

### ISSUE THREAD

**Issue by louisgv** [Coopss/EMNIST#5]
Hello,

I would like to cross validate my loss and accuracy with other. Below is my result:

![image](https://user-images.githubusercontent.com/6723574/29151989-7bf8b9aa-7d39-11e7-9113-5fc5273ccc08.png)

**Comment by ryanjcooper** [comment]
Hi @louisgv, your results seem fairly consistent with my training/test accuracy (~86%). I will be working on a deep-model extension to this project this weekend (hopefully), and I will report my findings.

It seems that creators of this data set reported 69.71% ± 1.47% (see table 3 from [here](https://arxiv.org/pdf/1702.05373.pdf)) accuracy using an OPIUM-based model, so this method beats that already. I have no benchmark for state-of-the-art (I would assume that >95% could be achieved using DCNNs).

I will leave this thread open for people to report any model progress or accuracy improvements.

Thanks for your interest!

EDIT:
I also noticed that they reported a peak accuracy of 77.57% ± 0.08% on the by_class set using a deep OPIUM-based classifier, should be noted that they claim that they believe higher accuracy can be achieved.

**Comment by jamesyzhb** [comment]
why 
![image](https://user-images.githubusercontent.com/24502654/39825943-b15eec9e-53e5-11e8-99e7-e621d788117e.png)

**Comment by jamesyzhb** [comment]
thanks

**Comment by CharlesLiuyx** [comment]
@jamesyzhb I encountered the same error. How to fix it? Please help

**Comment by ryanjcooper** [comment]
Hello @jamesyzhb, please open a new issue on the repository, but I believe it is an issue with a version mismatch with this code and Keras. What version of Keras and TensorFlow/Theano are you using?

**Comment by jamesyzhb** [comment]
Is a FLask version issue, thanks for the reminder
